x-pack/plugins/fleet/server/services/epm/elasticsearch/template/__snapshots__/template.test.ts.snap

```diff
@@ -834,45 +857,6 @@ exports[`EPM template tests loading system.yml: system.yml 1`] = `
             }
           }
         },
-        "network_summary": {
-          "properties": {
-            "ip": {
-              "properties": {
-                "*": {
-                  "type": "object"
-                }
-              }
-            },
-            "tcp": {
-              "properties": {
-                "*": {
-                  "type": "object"
-                }
-              }
-            },
-            "udp": {
-              "properties": {
-                "*": {
-                  "type": "object"
-                }
-              }
-            },
-            "udp_lite": {
-              "properties": {
-                "*": {
-                  "type": "object"
-                }
-              }
-            },
-            "icmp": {
-              "properties": {
-                "*": {
-                  "type": "object"
-                }
-              }
-            }
-          }
-        },
```

Comment:
Examples of incorrect mappings generated for names with wildcards, they won't be generated anymore.

Reply:
TODO: Investigate where this mapping came from, as no mapping is being generated now for this `network_summary`.